### ISSUE TITLE
Strange output to console when running code from the console

### ISSUE THREAD

**Issue by sov1178** [microsoft/vscode#75928]
- VSCode Version: 1.35.1
- OS Version: Windows 8 x64

Steps to Reproduce:
1. Open windows console
2. Run code using comand similar to: 
`code test.c`

The strings appear from time to time on the console from which code was started:
```
[main 2019-06-22T06:20:22.634Z] update#setState checking for updates
[main 2019-06-22T06:20:23.044Z] update#setState idle
```

I did search for the same issues, read those topics and checked:
1. I have correct PATH settings (it points to VSCode/bin folder). 
2. I am **absolutely** sure `code.cmd` (not `code.exe`) is run (I added a line which prints test string to `code.cmd` and see its output when starting VSCode from the command line).

**Comment by joaomoreno** [comment]
Hm... Which console is it?

**Comment by sov1178** [comment]
@joaomoreno Oh... me bad - it was not usual windows console (I thought console type was not so critical). I used to use the program FAR (https://www.farmanager.com/). Today, after I received your message, I tried to run VSCode from the usual cmd.exe console, and there is no annoying messages. But, when I run VSCode from the FAR the messages are still there. It is interesting, cause FAR runs cmd.exe to execute code.cmd). 

Any ideas how to fix it?
